lib/Traits/TraitsClasses/Table/ColumnTraits.ts

```diff
@@ -59,6 +59,7 @@ export default class TableColumnTraits extends ModelTraits {
       "The type of the column. If not specified, the type is guessed from " +
       "the column's name and values. Valid types are:\n\n" +
       Object.keys(TableColumnType)
+        .filter((type) => isNaN(Number(type)))
```

Comment:
Filter out numeric values from enum keys. Maybe those works, but they aren't useful for users as they will break much easier

Reply:
Perhaps that information should be put in a comment in the source code so it's easily available for future readers?

Reply:
I do not believe the comment will add any additional value; this was simply a note, so it is easier to review and clarify the reasoning behind the change.